acq4/pyqtgraph/metaarray/MetaArray.py

```diff
@@ -1113,14 +1112,17 @@ def writeHDF5(self, fileName, **opts):
             data[tuple(sl)] = self.view(np.ndarray)
             
             ## add axis values if they are present.
+            axKeys = ["values"]
+            axKeys.extend(opts.get("appendKeys", []))
             axInfo = f['info'][str(ax)]
-            if 'values' in axInfo:
-                v = axInfo['values']
-                v2 = self._info[ax]['values']
-                shape = list(v.shape)
-                shape[0] += v2.shape[0]
-                v.resize(shape)
-                v[-v2.shape[0]:] = v2
+            for key in axKeys:
+                if key in axInfo:
+                    v = axInfo[key]
+                    v2 = self._info[ax][key]
+                    shape = list(v.shape)
+                    shape[0] += v2.shape[0]
+                    v.resize(shape)
+                    v[-v2.shape[0]:] = v2
```

Comment:
Can you add here:

    else:
        raise TypeError('Cannot append to axis info key "%s"; this key is not present in the target file.' % key)